srml/sudo/src/lib.rs

```diff
@@ -37,6 +135,9 @@ decl_module! {
 	pub struct Module<T: Trait> for enum Call where origin: T::Origin {
 		fn deposit_event<T>() = default;
 
+		/// Allow the sudo key to make a `Root` call to a dispatchable function
```

Comment:
I interpret "allow" is meaning to give permission. This function not only allows a user to execute a function, it executes the given function.

Suggestion: "Authenticate a user and execute a dispatchable function."